canonical-web-and-design/vanilla-framework
# Fix table-related accessibility issues

## Done

Fixes https://github.com/canonical-web-and-design/vanilla-squad/issues/870

- Remove grid and table-related roles from `<table>` elements
- Add aria-labels to all tables

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/ table examples, ensure there are no visual or interaction changes
- Review whether aria-labels on tables are clear and understandable

Adding do not merge as this breaks the sortable table example which relies on roles. @bartaz what approach would you recommend to fix that?
